mappings/net/minecraft/loot/context/LootContext.mapping

```diff
@@ -76,3 +90,4 @@ CLASS net/minecraft/class_47 net/minecraft/loot/context/LootContext
 			METHOD write (Lcom/google/gson/stream/JsonWriter;Ljava/lang/Object;)V
 				ARG 1 writer
 				ARG 2 entity
+	CLASS class_8487 DirectEntry
```

Comment:
Don't think you need the `Direct` prefix; there's no "indirect entry", only `LootDataKey`.

Reply:
```suggestion
	CLASS class_8487 Entry
```